lib/rspec/rails/matchers/active_job.rb

```diff
@@ -15,6 +15,7 @@ def initialize(job)
             @args = []
             @queue = nil
             @at = nil
+            @with_block = nil
```

Comment:
You'd be better off calling this `@block` and setting it to a passed in block or an empty lambda.